data/createData.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
+require_once $_SERVER["DOCUMENT_ROOT"] . "/vendor/autoload.php";
+
+use TaskForce\app\parser\Parser;
+use TaskForce\app\parser\column\ColumnString;
+use TaskForce\app\parser\column\ColumnIntRand;
+use TaskForce\app\parser\column\ColumnIntOrder;
+use TaskForce\app\status\StatusNew;
+
+function datetimeToStr(int $rand) :string {
+    return date("Y-m-d h:m:s", $rand);
+}
+
+
+function createDataCategories($dir, $fileName) {
+    $Parser = new Parser($dir . "/" . $fileName);
+    $Parser->setTableName("category");
+
+    $Parser->pushColumnHandler(new ColumnString("name"));
+
+    $Parser->pushColumnHandler(new ColumnString("icon"));
+
+    $Parser->parseFile();
+}
+
+function createDataCities($dir, $fileName) {
+    $Parser = new Parser($dir . "/" . $fileName);
+    $Parser->setTableName("city");
+
+    $Parser->pushColumnHandler(new ColumnString("city"));
+
+    $Parser->pushColumnHandler(new ColumnString("lat"));
+
+    $Parser->pushColumnHandler(new ColumnString("long"));
+
+    $Parser->parseFile();
+}
+
+function createDataFeedback($dir, $fileName) {
+    $Parser = new Parser($dir . "/" . $fileName);
+    $Parser->setTableName("feedback");
+
+    $Parser->pushColumnHandler(new ColumnString("created_at"));
+
+    $Parser->pushColumnHandler(new ColumnString("rate"));
+
+    $descriptionHandler = new ColumnString("description");
+    $descriptionHandler->setMaxValueLen(255);
+    $Parser->pushColumnHandler($descriptionHandler);
+
+
+    $employerIdHandler = new ColumnIntRand("user_id");
+    $employerIdHandler->setMaxValue(20);
+    $Parser->pushColumnHandler($employerIdHandler);
+
+    $employerIdHandler = new ColumnIntRand("task_id");
+    $employerIdHandler->setMaxValue(10);
+    $Parser->pushColumnHandler($employerIdHandler);
+
+    $Parser->parseFile();
+}
+
+function createDataProfiles($dir, $fileName) {
+    $Parser = new Parser($dir . "/" . $fileName);
+    $Parser->setTableName("profile");
+
+    $Parser->pushColumnHandler(new ColumnString("address"));
+
+    $Parser->pushColumnHandler(new ColumnString("bd"));
+
+    $Parser->pushColumnHandler(new ColumnString("about"));
+
+    $Parser->pushColumnHandler(new ColumnString("phone"));
+
+    $Parser->pushColumnHandler(new ColumnString("skype"));
+
+    $Parser->parseFile();
+}
```

Comment:
В принципе, идея интересная. Давай попробуем немного переработать код, сделать его более читаемым. Например 
```suggestion
function createDataProfiles(string $directory, string $fileName): void
{
    $parser = new Parser($directory . '/' . $fileName);
    
    $parser
        ->setTableName('profile')
        ->pushColumnHandler(new ColumnString('address'))
        ......
        ->pushColumnHandler(new ColumnString('skype'))
    ;

    $parser->parseFile();
}
```